docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -76,7 +76,7 @@ test.beforeEach(async t => {
 Dummy data should be cleared after each test:
 
 ```js
-test.afterEach.always(async t => await User.remove());
+test.afterEach.always(async () => await User.remove());
```

Comment:
`async` and `await` are redundant here since you're returning a `Promise` from `User.remove()`.

Reply:
The explicitness doesn't hurt I think.

Reply:
There's no need for it and anyone using `xo` will need to remove it if they copy+paste. Better to just remove it and set an example by only shipping code that's linted even if it's only in the docs.